src/lib/y2users/linux/users_writer.rb

```diff
@@ -241,6 +260,17 @@ def edit_user(new_user, old_user, issues)
         log.error("Error modifying user '#{new_user.name}' - #{e.message}")
       end
 
+      # Edits the user's password
+      #
+      # @param new_user [User] User containing the updated information
+      # @param old_user [User] Original user
+      # @param issues [Y2Issues::List] a collection for adding an issue if something goes wrong
+      def edit_password(new_user, old_user, issues)
+        return if old_user.password == new_user.password
```

Comment:
Is it possible that the admin might want to change only the _other_ fields in `/etc/passwd` with this (real name etc.), but the password remains the same? In this case, those other changes would simply be ignored if the password is not also changed.

Reply:
The password is an object, and the equality is overloaded, see https://github.com/yast/yast-users/blob/master/src/lib/y2users/password.rb#L76. So, the passwords are not equal if any of the other fields is different.

Reply:
I don't think so. This guard clause applies only to the `edit_password` method. Or am I missing something?

Reply:
OK, thanks for the clarification.